### PR TITLE
start review

### PR DESCRIPTION
**Pull request by apatlpo**
will be updated soon

### CODE REVIEW

File: 1 DASK, with HPC cluster (PBS Pro).ipynb [hunk 1]
```diff
@@ -4,11 +4,24 @@
...
-    "# DASK using HPC cluster through PBS Pro batch queue system.\n",
-    "To use pangeo(http://pangeo.io), one need to get used to with some basic python packages used in pangeo.  DASK(http://dask.org) is one of those key component of pangeo.  Through DASK, you can parallelize your computation.  In this notebook, we use dask_jobqueue package based on PBS cluster implementation to show how one can make use. \n",
+    "**AP: change all 'you' in 'we'**\n",
     "\n",
-    "***For MPI users, this is like combination of writing code using mpi_init, submitting mpirun script to a cluster (without knowing what your context of code will be....)***\n",
+    "# DASK on HPC with PBS Pro batch queue system.\n",
     "\n",
+    "To use [pangeo](http://pangeo.io), one needs to get familiar with key python packages of its ecosystem.\n",
+    "[Dask](http://dask.org) is one of them which allow you to distribute your computations.\n",
+    "We'll focus here on distribution over an HPC cluster equipped with PBS.\n",
```
Comment (tinaok) [inline 1 DASK, with HPC cluster (PBS Pro).ipynb]:
```suggestion
    "We'll focus here on distribution over an HPC cluster equipped with PBS Pro.\n",
```

File: 1 DASK, with HPC cluster (PBS Pro).ipynb [hunk 2]
```diff
@@ -41,27 +54,37 @@
...
+    "\n",
+    "**AP: here and below, the use of the word 'chunk' is confusing here. Don't you mean 'smaller workers' or 'smaller parts of machine nodes?**\n",
     "\n",
-    "*cluster = PBSCluster(cores=1,memory='5 gb', walltime='1:00:00')*\n",
+    "`cluster = PBSCluster(cores=1,memory='5 gb', walltime='1:00:00')`\n",
     "\n",
     "ATT these chunks should be chosen well so that the chunk fits well to the cluster's pbs configuration. In this example it chose 'walltime 1 hour' since that is the max time limit of short quick job queue.\n",
     "\n",
-    "**In case you are using HPC cluster, if you make short, and small chunk, your job generally fits to some gap of non used resources of HPC cluster, thus it may start faster.  But this also creates small chunks of used resources, which makes it difficult to run big job for other users.  Thus it is important to consult with your HPC cluster managers.  **\n",
+    "**AP: I am not sure ATT is a standard acronym**\n",
```
Comment (tinaok) [inline 1 DASK, with HPC cluster (PBS Pro).ipynb:74]: https://writingexplained.org/english-abbreviations/attention

File: 1 DASK, with HPC cluster (PBS Pro).ipynb [hunk 3]
```diff
@@ -30,8 +43,8 @@
    "metadata": {},
    "source": [
     "---\n",
-    "## 2. Set up dask worker's configuration.  \n",
-    "This configuration is based on hal(CNES HPC Cluster), which use PBSPro as batch scheduler.   "
+    "## 2. Spawn a dask distributed cluster  \n",
```
Comment (tinaok) [inline 1 DASK, with HPC cluster (PBS Pro).ipynb]: keep set up here

File: 1 DASK, with HPC cluster (PBS Pro).ipynb [hunk 4]
```diff
@@ -431,7 +477,7 @@
    "metadata": {},
    "source": [
     "---\n",
-    "## 6. once you finish your computation, do not forget to stop your dask worker with following command.\n"
+    "## 6. once you're done computating, do not forget to stop your dask worker with following command.\n"
```
Comment (tinaok) [inline 1 DASK, with HPC cluster (PBS Pro).ipynb]: you mean computing? I can say 'finished your computation'..,

File: 3 zarr and re-chunking.ipynb [hunk 5]
```diff
@@ -4,24 +4,33 @@
...
-    "As seein in *'2. introduction to zarr'* notebook, zarr file have encodings called 'chunk'.  \n",
+    "# more zarr, rechunking\n",
+    "\n",
+    "As seein in [2. introduction to zarr](https://github.com/tinaok/Pangeo-for-beginners/blob/master/2%20Introduction%20to%20zarr.ipynb) notebook, zarr archives are made of 'chunks' which corresponds to files.\n",
     " \n",
     "How your data is 'chunk'ed plays important role on your computation time, through your cpu-time, and io-wait time.\n",
+    "For example:\n",
+    "\n",
+    "1. If chunk sizes are too small, calculations will create too many meta-data access and will slow down IO access. You may also slow down IO access of your colleagues.\n",
+    "2. If chunk order/size is not suitable to your computation, each dask workers need to exchange data by communication, thus your computational time gets slow.  \n",
```
Comment (tinaok) [inline 3 zarr and re-chunking.ipynb:15]: you -> we?

File: 3 zarr and re-chunking.ipynb [hunk 6]
```diff
@@ -4,24 +4,33 @@
...
     "\n",
-    "1. If your chunk size is too small, it will use too many meta-data access thus your IO access gets slow. (and you might even slow down IO access of your colleagues,,  )\n",
-    "2. If your chunk order/size is not suitable to your computation,  each dask workers need to exchange data by communication, thus your computational time gets slow.  \n",
     "\n",
-    "For example, you have a 3D data in $t$, $i,$ and $j$ coordinate $ f (t,i,j)$ then, you would like to make $f'(i,j)$ as\n",
+    "**AP: drawning would be better than equations to explain what is below**\n",
```
Comment (tinaok) [inline 3 zarr and re-chunking.ipynb:18]: I agree, need idea for a good figure

**Review by tinaok — CHANGES_REQUESTED** [review]
tt

File: 3 zarr and re-chunking.ipynb (hunk 5 above)
Comment (apatlpo) [reply 3 zarr and re-chunking.ipynb:15]: Il faut changer les you en we un peu partout, je te laisse faire ça.

File: 3 zarr and re-chunking.ipynb (hunk 6 above)
Comment (apatlpo) [reply 3 zarr and re-chunking.ipynb:18]: ok

**Pull request merged**